.github/actions/setup-polaris/get_polaris.sh

```diff
@@ -16,5 +16,5 @@ curl -LJ -o $TARGET_FILE $POLARIS_URL
 mkdir polaris
 tar -xzf $TARGET_FILE -C polaris
 rm $TARGET_FILE
-echo "polaris" >> $GITHUB_PATH
-echo "::set-output name=version::$INPUT_VERSION"
\ No newline at end of file
+echo "$(pwd)/polaris/polaris" >> $GITHUB_PATH
```

Comment:
🤔 is this the right path? Should `polaris` be duplicated?

Reply:
you are right, my commit points to the binary, not the directory